charts/datadog/templates/_containers-common-env.yaml

```diff
@@ -9,7 +9,7 @@
     secretKeyRef:
       name: {{ template "datadog.apiSecretName" . }}
       key: api-key
-{{- if .Values.datadog.remoteConfiguration.enabled }}
+{{- if or (.Values.datadog.remoteConfiguration.enabled .Values.remoteConfiguration.enabled) }}
```

Comment:
can't we set a variable to all these conditions with an `or` and use that variable?
One for the controller, and one for the agent. So we have a common global definition to use in these places

Reply:
Not a big fan of what this makes us do with the string comparison, but done in [20303ce](https://github.com/DataDog/helm-charts/pull/1058/commits/20303ce608848ebe087e9f6b489f2ed3138384ed).